JetStreamDriver.js

```diff
@@ -1733,68 +1732,68 @@ const BENCHMARKS = [
             , "./RexBench/UniPoker/benchmark.js"
         ],
         deterministicRandom: true,
-        testGroup: RexBenchGroup
+        tags: ["RexBench"],
     }),
     // Simple
     new DefaultBenchmark({
         name: "hash-map",
         files: [
             "./simple/hash-map.js"
         ],
-        testGroup: SimpleGroup
+        tags: ["Simple"],
     }),
     new AsyncBenchmark({
         name: "doxbee-promise",
         files: [
             "./simple/doxbee-promise.js",
         ],
-        testGroup: SimpleGroup,
+        tags: ["Simple",],
```

Comment:
Ah my previous nit was about `["Simple"]` vs `["Simple",]` but it's not a huge deal.

Reply:
ah this was easy to fix...